src/main/java/org/commonjava/util/partyline/PartylineException.java

```diff
@@ -0,0 +1,12 @@
+package org.commonjava.util.partyline;
+
+import java.io.IOException;
+
+public class PartylineException
+                extends IOException
+{
+    public PartylineException( String m )
```

Comment:
You'll need to use the constructor variant that includes a "cause" exception. That way we're wrapping the original and not losing it, which is CRITICAL.